models/ims-model.js

```diff
@@ -58,11 +58,12 @@ const setEformValue = (eform, fieldName, fieldValue) => {
 
 const setEformValues = (eform, caseRef) => {
   const today = new Date();
-  const time = today.getHours() + ':' + today.getMinutes();
+  //add 0 in front of single digit minutes
+  const time = today.getHours() + ':' + (today.getMinutes() < 10 ? '0' : '') + today.getMinutes();
 
   setEformValue(eform, 'caseid', caseRef);
-  setEformValue(eform, 'dtborec', today.toLocaleDateString());
-  setEformValue(eform, 'tmboec', time);
+  setEformValue(eform, 'dtBORec', today.toLocaleDateString("en-GB"));
+  setEformValue(eform, 'tmBORec', time);
```

Comment:
per https://collaboration.homeoffice.gov.uk/jira/browse/PAF-299 this looks right.